assets/src/components/template-inserter/index.js

```diff
@@ -102,16 +102,20 @@ class TemplateInserter extends Component {
 											className="amp-stories__blank-page-inserter editor-block-preview__content block-editor-block-preview__content editor-styles-wrapper"
 										/>
 									</div>
-									{ storyTemplates && storyTemplates.map( ( item ) =>
-										<BlockPreview
+									{ storyTemplates && storyTemplates.map( ( item ) => (
+										<Button
```

Comment:
@swissspidy There's an issue with wrapping this inside a button -- if the block that's being previewed includes a button then this means a button within a button which is not allowed.

Reply:
Yes, I saw your comment in the previous file. `core/button` is not in `ALLOWED_CHILD_BLOCKS` though, so I am not sure in which scenario this could happen.

Reply:
I think there's an inserter button or something like within a Page block which is also included in the preview.

Reply:
Ah, I see! This reminds me of https://github.com/WordPress/gutenberg/pull/14241 where I previously suggested adding an option to disable the inserter/appender completely. Once again proof that such a feature is needed. I hope I can manage to work on a PR for that soon.

Reply:
@swissspidy: I'm seeing these errors now when using templates, is this something you were planning working on soon? Otherwise, perhaps we should revert using the `button` element for now?
![Screen Shot 2019-04-02 at 7 44 55 PM](https://user-images.githubusercontent.com/3294597/55424154-ec966e80-557f-11e9-9f70-a0c0ada1c1f7.png)

Reply:
Yes, it's on my list, although I don't see it as a pressing issue. Fixing it upstream takes some time anyway as we'd have to wait for the next Gutenberg release in 1-2 weeks.

A workaround could be the remove the DOM node manually, but I'm not sure that'd be worth the effort.